pkg/storage/unified/apistore/watcher_test.go

```diff
@@ -151,12 +154,16 @@ func TestDeleteTriggerWatch(t *testing.T) {
 	storagetesting.RunTestDeleteTriggerWatch(ctx, t, store)
 }
 
-func TestWatchFromZero(t *testing.T) {
-	ctx, store, destroyFunc, err := testSetup(t)
-	defer destroyFunc()
-	assert.NoError(t, err)
-	storagetesting.RunTestWatchFromZero(ctx, t, store, nil)
-}
+// Not Supported by unistore because there is no way to differentiate between:
+// - SendInitialEvents=nil && resourceVersion=0
+// - sendInitialEvents=false && resourceVersion=0
+// This is a Legacy feature in k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go#196
+// func TestWatchFromZero(t *testing.T) {
+// ctx, store, destroyFunc, err := testSetup(t)
+// defer destroyFunc()
+// assert.NoError(t, err)
+// storagetesting.RunTestWatchFromZero(ctx, t, store, nil)
+// }
```

Comment:
Praise, we should probably not support this. Adding a "Watch from zero" section to mention this in [Unified Storage: Kubernetes Compatibility Notes](https://docs.google.com/document/d/1w72R_RYZl445KV8c_DRhB-KcTG4bs1SFVlmxz5ZmY3w/edit#heading=h.magfjub697rf). Feel free to change it as needed.